src/Tribe/Template.php

```diff
@@ -533,6 +533,39 @@ public function template( $name, $context = array(), $echo = true ) {
 		// Setup the Hook name
 		$hook_name = implode( '/', $namespace );
 
+		/**
+		 * Allow users to filter the HTML before rendering
+		 *
+		 * @since  TBD
+		 *
+		 * @param string $html      The initial HTML
+		 * @param string $file      Complete path to include the PHP File
+		 * @param array  $name      Template name
+		 * @param self   $template  Current instance of the Tribe__Template
+		 */
+		$pre_html = apply_filters( 'tribe_template_pre_html', null, $file, $name, $this );
+
+		/**
+		 * Allow users to filter the HTML by the name before rendering
+		 *
+		 * E.g.:
+		 *    `tribe_template_pre_html:events/blocks/parts/details`
+		 *    `tribe_template_pre_html:events/embed`
+		 *    `tribe_template_pre_html:tickets/login-to-purchase`
+		 *
+		 * @since  TBD
+		 *
+		 * @param string $html      The initial HTML
+		 * @param string $file      Complete path to include the PHP File
+		 * @param array  $name      Template name
+		 * @param self   $template  Current instance of the Tribe__Template
+		 */
+		$pre_html = apply_filters( "tribe_template_pre_html:$hook_name", $pre_html, $file, $name, $this );
```

Comment:
:warning: Words in hook names should be separated using underscores. Expected: "tribe_template_pre_html_$hook_name", but found: "tribe_template_pre_html:$hook_name".

<sub>hash: 2143604a63324bbe1e4a6aff60f03631</sub>

Reply:
Betting you can avoid this by using `'tribe_template_pre_html:' . $hook_name`